ibft/storage/store.go

```diff
@@ -173,8 +173,7 @@ func (i *ibftStorage) CleanAllInstances(msgID []byte) error {
 	if err != nil {
 		return errors.Wrap(err, "failed to remove decided")
 	}
-	i.logger.Debug("removed decided", zap.Int("count", n),
-		zap.String("identifier", hex.EncodeToString(msgID)))
+	i.logger.Debug("removed decided", zap.Int("count", n), logging.IdentifierBytes(msgID))
```

Comment:
why not IdentifierString?

Reply:
the naming convention i used was based on the data type of the function argument. Because the messageID here is a byte slice, the name was made IdentifierBytes. 

if we have a naming convention that is output based, like we name it based on the output type of the function, we will run into the issue where names collide 

```go

func IdentifierString(id []bytes) zap.Field {
  return a string of hex encoded bytes 
} 


func IdentifierString(id string) zap.Field {
  return the id which is already a string
} 

```